src/declarations/backend/backend.did

```diff
@@ -1,7 +1,27 @@
-type anon_class_20_1 = 
```

Comment:
neviem či by som verzovala tieto did súbory, keďže sú generované, ako je to v ostatných projektoch na dfinity?